src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -80,6 +86,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
         $filenames = $input->getArgument('filename');
         $showDeprecations = $input->getOption('show-deprecations');
+        $this->format = $input->getOption('format');
+
+        if ('github' === $this->format && !class_exists(GithubActionReporter::class)) {
```

Comment:
Since you bumped the min console version in the bridge, this code and other related stuff about `GithubActionReporter` detection is now unnecessary